### ISSUE TITLE
PER WORLD SETTINGS DOESN'T WORK

### ISSUE THREAD

**Issue by Kloverfist** [NathanCollaert/RewardsLite#44]
In ([https://github.com/NathanCollaert/RewardsLite/blob/master/src/resources/default-reward.yml]) world settings about playtime is 
```disableGainingPlaytimeInWorlds: 
  - "world"```
doesn't matter how many times, if i set worlds here, iwhen i reload, it will delete my world settings and replace it with this:

`disableGainingchePlaytimeInWorlds: []`

PLEASE HELP :C

**Comment by NathanCollaert** [comment]
Hi there, any errors on startup?

**Comment by Kloverfist** [comment]
> Hi there, any errors on startup?

nope

**Comment by NathanCollaert** [comment]
Can you provide me with the file from before and after a restart please? With the changes made you want to apply in the before file.

**Comment by Klover91** [comment]
[RANKUP.zip](https://github.com/NathanCollaert/RewardsLite/files/10523040/RANKUP.zip)

**Comment by NathanCollaert** [comment]
Hey there @Klover91, I just checked and this is indeed a bug. I'll try to fix this asap.

**Comment by Klover91** [comment]
> 

thank you very much! my whole gameplay is base on the correct function of your plugin an i will open very soon to public🙏 🙏

**Comment by Klover91** [comment]
can you tell me pretty much how long it will take to fix this?

**Comment by NathanCollaert** [comment]
Hi there @Klover91, I will be fixing this bug tonight.

**Comment by Kloverfist** [comment]
amazing ! thank you very much! <3

**Comment by NathanCollaert** [comment]
This has been fixed in v3.1.4.
